### PR TITLE
windows: allow to specify pkgType without overriding the name

### LINKED ISSUE #12: Windows upgrade path will not work with .exe installers
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

Right now, the windows script fetches the latest version from the prod repo, but the file it searches for is hardcoded to `.msi`. This will cause the upgrade check to be skipped for integrations that use the `.exe` format

Action should get a new input, e.g. `installerType`, which defaults to `msi` and can be specified to be `exe`. This can be appended to the package names in the action, and would fix the issue as well as simplify the usage for exe installers (which right now have to specify a full name override).

### PR DESCRIPTION
**Pull request by roobre**
Fixes #12

### CODE REVIEW

File: windows/test_msi.ps1 [hunk 1]
```diff
@@ -3,43 +3,60 @@ param (
     [string]$ARCH = "amd64",
     [string]$TAG = "v0.0.0",
     [string]$UPGRADE = "false", # upgrade: upgrade msi from last released version.
+    [string]$PKG_TYPE = "msi",
```
Comment (davidbrota) [inline windows/test_msi.ps1:6]: you could use the ValidateSet here https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.1#validateset-attribute

Comment (davidbrota) [reply windows/test_msi.ps1:6]: don't need to change it now

File: windows/test_msi.ps1 [hunk 2]
```diff
@@ -48,27 +65,34 @@ if ($UPGRADE -eq "true")
...
+    }
+    else
+    {
+        $PKG_NAME = "${INTEGRATION}-${ARCH}-installer.${version}.exe"
+    }
 }
-$PKG_LATEST_NAME = "${PKG_DIR}\${PKG_NAME}"
-write-host "::group::ℹ️ Installing generated msi: ${PKG_LATEST_NAME}"
-if ($PKG_LATEST_NAME -notlike "*.msi")
+$PKG_PATH = "${PKG_DIR}\${PKG_NAME}"
```
Comment (davidbrota) [inline windows/test_msi.ps1:87]: joining paths is safer with https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/join-path?view=powershell-7.1

don't need to change it now

**Review by davidbrota — APPROVED** [review]

File: windows/test_msi.ps1 (hunk 1 above)
Comment (roobre) [reply windows/test_msi.ps1:7]: This looks great! I suck at powershell.

File: windows/test_msi.ps1 (hunk 2 above)
Comment (roobre) [reply windows/test_msi.ps1]: Done!

**Pull request merged**